setup.py

```diff
@@ -67,7 +66,11 @@ def package_files(directory):
         "console-progressbar==1.1.2",
         "ansicolors==1.1.8",
```

Comment:
Could we also not specify exact version?

Reply:
@jondequinor has quite good reasons for this approach, namely that we can't vet every release, especially for small projects we depend on. I tend to agree with that. For larger projects like `numpy` and `pandas` this would not apply.

Reply:
Alright, I agree with that reasoning!